### PR TITLE
Story/vethub 47

### PR DESCRIPTION
**Pull request by Jo-Zh**
to test:
`yarn test`
to see UI:
`yarn dev`

### CODE REVIEW

File: src/types/MCQ.d.ts [hunk 1]
```diff
@@ -7,6 +7,7 @@ export interface MCQuestion {
   statement: string;
   tags?: string[];
   optionsList: MCQOptions[];
+  link?: string;
```
Comment (anzau) [inline src/types/MCQ.d.ts]: I don't think we should make this optional.

**Review by anzau — APPROVED** [review]

**Comment by anzau** [comment]
Just one comment, other than that all good.

File: src/components/MCQ/MCQQuestion.vue [hunk 2]
```diff
@@ -29,14 +29,18 @@
...
+import type { MCQOptions } from "@type/MCQ.d.ts";
 import MCQOption from "./MCQOption.vue";
 import MCQButton from "./MCQButton.vue";
 import { useQuizStore } from "@/store/QuizStore";
 
 const statUpdate = useQuizStore();
 
-const { statement, optionsList, _id } = defineProps<MCQuestion>();
+const { statement, optionsList, _id } = defineProps<{
+  statement: string;
```
Comment (anzau) [inline src/components/MCQ/MCQQuestion.vue]: Make this a type/interface please

Comment (Jo-Zh) [reply src/components/MCQ/MCQQuestion.vue]: fixed

**Review by anzau — APPROVED** [review]

**Pull request merged**